src/components/Modal/index.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './Modal.scss';
+import { childrenPropType } from "../../utils/propTypes";
 
 const Modal = ({
-  id, children, title,
+  id, children, title, bottomSheet, fixedFooter, footer,
```

Comment:
Separate to new lines for better readability

Reply:
I have added each of the `props` into a new line to improve readability.